src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -72,9 +72,10 @@
     private static final int SLOPEOVERRIDE_FLAG_INDEX = 5;
     private static final int WHEELCHAIR_ACCESSIBLE_FLAG_INDEX = 6;
     private static final int BICYCLE_NOTHRUTRAFFIC = 7;
+    private static final int WALK_NOTHRUTRAFFIC = 8;
 
     /** back, roundabout, stairs, ... */
-    private byte flags;
+    private short flags;
```

Comment:
This will increase the size of the graph. Do you have an idea by how much?

Reply:
I guess less than 5%, maybe 2-3%. Anyway, we do not have a  choice. Walk nothru bit must be added.

Reply:
I did some tests and it seems that pruning plays insignificant role in the memory use of graph building process.  There are other stages which consume much more memory and pruning will not be the bottleneck. So, in my opinion, the accurate pruning should be a standard feature.